bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -5316,21 +5327,15 @@ PreparedStatement createInsertArtifactStatement(int artifact_type_id, long obj_i
 
 	@Deprecated
 	BlackboardArtifact newBlackboardArtifact(int artifact_type_id, long obj_id, String artifactTypeName, String artifactDisplayName, long data_source_obj_id, CaseDbConnection connection) throws TskCoreException {
-		acquireSingleUserCaseWriteLock();
+		BlackboardArtifact.Type type = getArtifactType(artifact_type_id);
```

Comment:
This is not public API, do we even need this or can it just be deleted?

Reply:
@rcordovano This method is used by other versions of newBlackboardArtifact. It seems to be a helper method, I have added documentation and made the method private.